setup.py

```diff
@@ -6,7 +6,6 @@
 
 """
 Run with:
-
```

Comment:
@manasRK I still see many such deleted lines in the diff (top tab, "Files changed").